src/input/code_templates/c/static_code/exi_basetypes.c.jinja

```diff
@@ -134,6 +134,24 @@ int exi_basetypes_convert_64_from_signed(const exi_signed_t* exi_signed, int64_t
     return res;
 }
 
+static void _reverse_array(uint8_t* data, size_t data_size)
+{
+    if (!data || !data_size)
```

Comment:
If this function is static and you control the inputs, why should it have such a silent error handling of data? It will have gone wrong in `exi_basetypes_convert_bytes_from_unsigned` before this function is reached.

data_size makes sense of course.

Reply:
That's a good point. I was sort of "safe programming".

This code isn't exactly a hot path though.